packages/thumbprint-react/components/Chip/index.tsx

```diff
@@ -34,7 +40,10 @@ export default function FilterChip({
             onClick={onClick}
             aria-pressed={isSelected}
         >
-            {text}
+            <div className="flex items-center">
```

Comment:
This might seem counter-intuitive, but we actually don't use Atomic CSS classes in Thumbprint React components.

If we were to use them in Thumbprint React, we'd be creating an implicit dependency between the version of Thumbprint React and Thumbprint Atomic that a codebase is using.

For example, imagine:

1. We add a class called `sr-only` to Atomic which hides elements from everything except a screenreader.
2. Thumbprint React starts using `.sr-only`.
3. Website upgrades to the latest version of Thumbprint React but doesn't upgrade Thumbprint Atomic.

We'd then end up in a situation where the class is used in Thumbprint React but isn't loaded on the page.

Alternatively we could import Thumbprint Atomic _within_ Thumbprint React. But that causes a new problem where the Atomic classes get loaded multiple times on a page.

Reply:
Ahh, this makes sense. Moved the styling to the scss file